sjb/generated/test_branch_jenkins_client_plugin.xml

```diff
@@ -165,7 +165,7 @@ for image in &#39;registry.svc.ci.openshift.org/ci/clonerefs:latest&#39; &#39;re
         fi
     done
 done
-clonerefs_args=\${CLONEREFS_ARGS:---repo=openshift,openshift-ansible=master --repo=openshift,aos-cd-jobs=master }
+clonerefs_args=\${CLONEREFS_ARGS:---repo=openshift,openshift-ansible=master --repo=openshift,aos-cd-jobs=master --repo=openshift,origin=release-3.11 }
```

Comment:
It still uses openshift-ansible's master branch

Reply:
ah ... good catch / thanks @vrutkovs 

I'll try adding an entry in the sync block for openshift-ansible

Reply:
looks like that did the trick @vrutkovs